lib/shared/store.js

```diff
@@ -41,7 +41,8 @@ const DEFAULT_STATE = Immutable.fromJS({
   flashResults: {},
   flashState: {
     percentage: 0,
-    speed: 0
+    speed: null,
+    humanSpeed: ''
```

Comment:
I'm using empty string to simplify the logic here, and since empty string is the `null` of its respective domain I it makes sense. The only reason we need `null` and can't only use `0` in `speed` is because zero might actually be hit with certain devices during flashing so it can't represent a value responsible for removing the flashing status subtitle with the speed and ETA... maybe we can use the flashing flag instead, however.

@jhermsmeier when we hit `0` speed I assume we should replace the status with something more helpful than simply "0 B/s" – e.g. "Unknown speed and ETA" or "Speed and ETA unavailable", whatever is more succinct to the user.

Reply:
I guess I should clarify that I was only asking about the `humanSpeed` property - in `lib/shared/store.js` it defaults to `''` whereas in `lib/shared/models/flash-state.js` it defaults to `null`, and I was asking if they should be using the same default value for consistency purposes.